recipe/meta.yaml

```diff
@@ -37,7 +37,7 @@ requirements:
   run:
     - python
     - cloudpickle >=1.6.0,<=3.0.0,!=2.1.0,!=2.2.0
-    - snowflake-connector-python >=3.14.0,<4.0.0
+    - snowflake-connector-python >=3.12.0,<4.0.0
```

Comment:
why this one doesn't match the one in setup.py?

Reply:
3.14.0 is not available is anaconda and is causing build failures. That's why we relaxed constraints here.